lib/src/room.dart

```diff
@@ -396,17 +404,6 @@ class KurentoRoom extends ChangeNotifier implements Room {
 
       case ParticipantMessageType.INCOMING_TRACK_CREATE:
         await _createIncomingTrack(participantMessage.trackId);
-
-        if (serviceRequestState == ServiceRequestState.queued) {
-          // HACK: If the Agent is sending audio and we still think we're queued, we're not receiving messages on the
-          // service request presence topic. Until we can figure out why this is happening, pretend we received a
-          // message and transition the service request status to started.
-          _log.shout('missed service request status message topic=$_serviceRequestPresenceTopic');
-          _agentName = '';
-          _serviceRequestState = ServiceRequestState.started;
-          await _updateParticipantStatus();
-          notifyListeners();
-        }
```

Comment:
This was buggy for a couple of reasons:
1. We would transition from `queued` to `started` and skip `assigned`, so if an app has any logic that triggers on `assigned` (which the Aira Explorer app currently does), it wouldn't execute.
2. We didn't have the Agent name, so if an app has any logic that triggers on the presence of the Agent name (which the Aira Explorer app currently does), it wouldn't execute.